### DISCUSSION TITLE
Как настроить 4k?

### DISCUSSION THREAD

**Discussion by kinopub** [quarckster/kodi.kino.pub#166] [Q&A] [answered]
Вопрос по плагину кинопаб 2.12.4 к коди 18.9 на андроид 9.0 телек Sony x3 9505

В настройках плагина есть выбор качества видео 4к, но при воспроизведении максимум что даёт 1920.

Подскажите в  какую сторону кAпать?

**Comment by quarckster** [comment, marked as answer]
Попробуйте на сайте в "Настройках устройства" поставить галочку напротив "Поддержка 4K".

**Comment by kinopub** [comment]
Супер! 4к есть 

блин но теперь пропал 5.1 звук :)

**Reply by quarckster** [reply]
Плагин к этому отношения не имеет.

**Reply by kinopub** [reply]
Все верно. Простите за невнимательность. Не выбрал настройки на телеке куда пускать звук (телек/ресивер) 

Все опции должны быть выбраны или только 4к достаточно? 

![71F67A9A-7E5E-4753-BD33-75F99921A4AF](https://user-images.githubusercontent.com/78928879/107701181-36deef80-6cc1-11eb-9627-a6abbd00c969.jpeg)
